keyboards/handwired/dygma/raise/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = STM32F411
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
```

Comment:
We do highly recommend enabling this, since it adds a way to get into the bootloader to reflash, if a reset button isn't readily accessible.